client/components/tinymce/plugins/contact-form/dialog/field-edit-button.jsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import Popover from 'components/popover';
+
+export default React.createClass( {
+	displayName: 'ContactFormDialogFieldEditButton',
+
+	mixins: [ PureRenderMixin ],
+
+	propTypes: {
+		expanded: PropTypes.bool.isRequired
+	},
+
+	getInitialState: function() {
+		return {
+			showTooltip: false
+		};
+	},
+
+	render() {
+		const classes = classNames("editor-contact-form-modal-field__edit", {
```

Comment:
We need another [space](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#arrays-and-function-calls) in:
```javascript
const classes = classNames( "editor-contact-form-modal-field__edit",
```

You can also see relevant style warnings locally by running
```
make eslint-branch 
```
Feel free to ignore existing warnings outside of the new `field-edit-button.jsx`